modules/installation-downloading-image-restricted.adoc

```diff
@@ -36,12 +37,41 @@ You must download images with the highest version that is less than or equal to
 the {product-title} version that you install. Use the image versions that match
 your {product-title} version if they are available.
 ====
-
+endif::vsphere[]
 ifdef::osp[]
 . Download the *{op-system-first} - OpenStack Image (QCOW)* image.
 endif::osp[]
 ifdef::vsphere[]
-. Download the *{op-system-first} - vSphere* image.
+. Download the {op-system-first} OVA image by completing the following steps: 
```

Comment:
Maybe some of these steps should closely align with the [bare metal restricted environment](https://docs.openshift.com/container-platform/4.14/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html#installation-user-infra-machines-iso_installing-restricted-networks-bare-metal) steps?